plugins/soundsourcemediafoundation/soundsourcemediafoundation.cpp

```diff
@@ -478,7 +478,16 @@ bool SoundSourceMediaFoundation::configureAudioStream(const AudioSourceConfig& a
                 << "failed to get current media type from stream";
         return false;
     }
+    //------------ Get bitrate from the file, before we change it to get uncompressed audio
+    UINT32 aacbitrate;
+    hr = pAudioType->GetUINT32(MF_MT_AUDIO_AVG_BYTES_PER_SECOND, &aacbitrate);
+    if (FAILED(hr)) {
+        qWarning() << kLogPreamble << "error getting MF_MT_AUDIO_AVG_BYTES_PER_SECOND with pAudioType";
+    }
+    setBitrate(aacbitrate*8/1000);
```

Comment:
Use some space ;) I would also add braces "(avgBytesPerSecond * 8) / 1000" to emphasize that the multiplication must precede the integer division.